.github/workflows/docs.yml

```diff
@@ -13,4 +13,6 @@ jobs:
         with:
           python-version: '3.x'
       - name: Run ansible-doc-test
-        run: ANSIBLE_LIBRARY="." python utils/ansible-doc-test roles plugins
+        run: |
+          python -m pip install ansible
```

Comment:
This will force the use of the latest ansible version for the test. Can you install ansible only if it is not installed yet?